packages/editor/src/browser/editor-quick-open-service.ts

```diff
@@ -0,0 +1,154 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { inject, injectable } from 'inversify';
+import {
+    QuickOpenModel, QuickOpenItem, QuickOpenMode, PrefixQuickOpenService,
+    OpenerService, QuickOpenGroupItemOptions, QuickOpenItemOptions,
+    QuickOpenHandler, QuickOpenOptions
+} from '@theia/core/lib/browser';
+import URI from '@theia/core/lib/common/uri';
+import { LabelProvider } from '@theia/core/lib/browser/label-provider';
+import { WorkspaceService } from '@theia/workspace/lib/browser/workspace-service';
+import { Command } from '@theia/core/lib/common';
+import { CancellationTokenSource } from '@theia/core/lib/common';
+import { EditorManager } from './editor-manager';
+import { EditorWidget } from './editor-widget';
+
+export const SHOW_ALL_OPENED_EDITORS: Command = {
+    id: 'workbench.action.showAllEditors',
+    category: 'View',
+    label: 'Show All Opened Editors'
+};
+
+@injectable()
+export class EditorQuickOpenService implements QuickOpenModel, QuickOpenHandler {
+
+    @inject(OpenerService)
+    protected readonly openerService: OpenerService;
+
+    @inject(PrefixQuickOpenService)
+    protected readonly prefixQuickOpenService: PrefixQuickOpenService;
+
+    @inject(LabelProvider)
+    protected readonly labelProvider: LabelProvider;
+
+    @inject(EditorManager)
+    protected readonly editorManager: EditorManager;
+
+    @inject(WorkspaceService)
+    protected readonly workspaceService: WorkspaceService;
+
+    readonly prefix: string = 'edt ';
+
+    get description(): string {
+        return 'Show All Opened Editors';
+    }
+
+    getModel(): QuickOpenModel {
+        return this;
+    }
+
+    getOptions(): QuickOpenOptions {
+        return {
+            placeholder: '',
```

Comment:
I don't think we need the `placeholder` value here, the documentations states that:

https://github.com/theia-ide/theia/blob/9395eda975d3c76557a1c36c385ef3e22c3e62e8/packages/core/src/browser/quick-open/prefix-quick-open-service.ts#L64